### ISSUE TITLE
help

### ISSUE THREAD

**Issue by Lm0611** [wurenkai/UltraLight-VM-UNet#33]
Download the ISIC 2017 train dataset from [this](https://challenge.isic-archive.com/data) link and extract both training dataset and ground truth folders inside the /data/dataset_isic17/.
请问这个是要怎么操作的，请详细一点告诉我

**Comment by wurenkai** [comment]
Hi, you can select to the 2017 dataset from the official website and download the 2000 ‘Training Data’ images provided by the official website as the ISIC2017 dataset used. After downloading, you then extract the corresponding original images and segmentation masks inside.

**Comment by Lm0611** [comment]
![image](https://github.com/wurenkai/UltraLight-VM-UNet/assets/162393396/c1e3761b-99ad-48cc-b239-ac28a2bc7999)
为什么数据集下载之后，mask是这样的呢

**Comment by wurenkai** [comment]
The “_superpixers” are not required. Masks need to be downloaded from “Training Ground Truth”.
![image](https://github.com/wurenkai/UltraLight-VM-UNet/assets/124028634/efe9b034-1261-449b-9f2b-4d0e866418dd)
